powdr_cli/src/main.rs

```diff
@@ -480,3 +500,44 @@ fn optimize_and_output<T: FieldElement>(file: &str) {
         pilopt::optimize(compiler::analyze_pil::<T>(Path::new(file)))
     );
 }
+
+#[cfg(test)]
+mod test {
+
+    use backend::Backend;
+    use tempfile;
+
+    use crate::{run_command, Commands, CsvRenderMode, FieldArgument};
+    #[test]
+    fn test_simple_sum() {
```

Comment:
Nice! We should actually have more CLI tests like this for the other commands as well (unrelated to this PR ofc)